tests/unittest/disaggregated/test_cache_transceiver_single_process.py

```
@@ -1493,6 +1510,95 @@ def test_cache_transceiver_v1_dsa_indexer(
+    # Deliberately NOT setting TLLM_LOG_LEVEL_BY_MODULE: bounce engagement is verified
+    # programmatically inside the child (agent.bounce_enabled / bounce_submit_count),
+    # and a non-empty per-module level map can hang the child at exit (static-destruction
+    # order: CudaMemPool's deleter logs through an already-destroyed Logger module map).
+    env.pop("UCX_NET_DEVICES", None)
+    # Do not override TRTLLM_NIXL_BOUNCE_DISABLE_FABRIC_MEMORY: the test must
+    # exercise the production default, including automatic cudaMalloc fallback
+    # on devices without fabric-memory support.
+    env.pop("TRTLLM_NIXL_BOUNCE_DISABLE_FABRIC_MEMORY", None)
+
+    node_id = f"{__file__}::{request.node.name}"
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)